source/administration/object-management.rst

```diff
@@ -83,23 +102,31 @@ MinIO supports both :ref:`duration based object retention <minio-object-locking-
    :align: center
    :width: 600px
 
-See :ref:`minio-object-locking` for more complete documentation.
+Delete operations against a WORM-locked object depend on the specific operation:
+
+- Delete operations which do not specify a version ID result in the creation of a "Delete Marker"
+- Delete operations which specify the version ID of a locked object result in an locking error
+
+You *must* enable object locking during bucket creation.
```

Comment:
Maybe this is covered in another part of the document, but just seeing this line by itself suggests object locking is required everywhere whether you want it or not. Perhaps somethng like "it may not be added to an existing bucket" at the end.

Reply:
Tweaked, it is in fact just at bucket creation that you even have the option.